src/commcare_cloud/ansible/deploy_logstash.yml

```diff
@@ -5,7 +5,7 @@
   vars:
     sansible_logstash_family: 7.x
     sansible_java_version: 8
-    sansible_java_apt_keyserver: 'http://keys.gnupg.net'
+    sansible_java_apt_keyserver: 'https://keyserver.ubuntu.com'
```

Comment:
There has been a mention on the #devops slack channel regarding the host `keys.gnupg.net` being down for some while. I had to replace the keyserver host with this one for it to work. 

Why `keyserver.ubuntu.com`?
No real reason other than the fact that it's a different keyserver. We might go with `https://pgp.mit.edu/` also.

That said, I'm not sure if this is what we want to do, ie permanently changing the keyserver (until it's down, maybe changing it again, etc). I'm not ansible-smart, but is there a way we could try a fallback keyserver in case the one fails? 

Open to suggestions...